src/utilities/processo-juridico/index.ts

```diff
@@ -21,3 +21,34 @@ export function format(processoJuridico: string) {
       return result;
     }, '');
 }
+
+export function verifyDigit(processo: string): boolean {
+  const digits = processo.split('');
+  const verificationDigits = digits.splice(7, 2).join('');
+
+  const digits1to11 = digits.slice(0, 11).reduce((acc, digit, index) => {
+    return acc + +digit * Math.pow(10, 10 - index);
+  }, 0);
+
+  const firstRemainder = digits1to11 % 97;
+
+  const digits12to18 = digits.slice(11).reduce((acc, digit, index) => {
+    return acc + +digit * Math.pow(10, 6 - index);
+  }, 0);
+
+  const secondRemainder = (firstRemainder * 1_000_000_000 + digits12to18 * 100) % 97;
+
+  let verifier = 98 - secondRemainder;
```

Comment:
Why you use `const` for other variables and here you use `let`?

Reply:
Oops, just changed it